test/functionalTest/cases/0000_ngsild/ngsild_entity_types.test

```diff
@@ -239,18 +251,24 @@ Date: REGEX(.*)
 08. GET Entity Types - see five types
 =====================================
 HTTP/1.1 200 OK
-Content-Length: 26
+Content-Length: 136
 Content-Type: application/json
 Link: <https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"
 Date: REGEX(.*)
 
-[
-    "T0",
-    "T1",
-    "T2",
-    "T3",
-    "T4"
-]
+{
+    "id": "urn:ngsi-ld:EntityTypeList:REGEX(.*)",
+    "type": "EntityTypeList",
+    "typeList": [
+        [
```

Comment:
Why a double array?

The section 5.2.24 of the NGSI-LD API spec says that typeList is a array of strings, not an array of arrays of strings ...

Reply:
Fixed in 068159b